Client/Impl/Builder/ZeebeClientBuilder.cs

```diff
@@ -111,19 +117,50 @@ public IZeebeClientFinalBuildStep UseAccessTokenSupplier(IAccessTokenSupplier su
 
         public IZeebeClientFinalBuildStep UseKeepAlive(TimeSpan keepAlive)
         {
-            this.keepAlive = keepAlive;
+            KeepAlive = keepAlive;
             return this;
         }
 
         public IZeebeClientFinalBuildStep UseRetrySleepDurationProvider(Func<int, TimeSpan> sleepDurationProvider)
         {
-            this.sleepDurationProvider = sleepDurationProvider;
+            SleepDurationProvider = sleepDurationProvider;
             return this;
         }
 
         public IZeebeClient Build()
         {
-            return new ZeebeClient(Address, Credentials, keepAlive, sleepDurationProvider, loggerFactory);
+            return new ZeebeClient(Address, Credentials, KeepAlive, SleepDurationProvider, LoggerFactory, Certificate);
+        }
+
+        private X509Certificate2 CreateFromPemFile(string fileName)
```

Comment:
:x: Feels like this shouldn't be part of this PR, is this another feature you want to get merged?

Reply:
Ok nevermind I have re-read our PR description:

Reply:
Yes this is the explanation.